commands-yml/parse.js

```diff
@@ -88,38 +83,35 @@ Handlebars.registerHelper('capitalize', (driverName) => {
     case 'uiautomator':
       return 'UiAutomator';
     default:
-      return driverName[0].toUpperCase() + driverName.substr(1);
+      return driverName.length === 0 ? driverName : driverName[0].toUpperCase() + driverName.substr(1);
```

Comment:
-> `length < 2 ? `

Reply:
If the length is '1' it won't be a problem because `driverName.substr(1)` will just return an empty string.